career-readiness.md

```diff
@@ -5,13 +5,17 @@ The LEARN program focuses on readying our students for a career in technology. T
 LinkedIn is the world's largest online professional networking community. An active presence on LinkedIn allows you to make professional connections, look for employment opportunities, connect with others who share your interests, and learn more about the tech industry.
 
 - Submit your "about me" statement:
+My name is Samuel LaBrutte, I am a military veteran who spent most of his time during his military career as a Civil Engineer. As a result, I have gained a lot of experience in critical areas such as: problem solving, team building and communication. The things I'm looking forward to the most in web developing is being able to see how well I can solve problems that arise. I'm also looking for to learn from both my peers and mentors about ways to solve any problem.
```

Comment:
Instead of saying looking forward to, I would say something like "what excited me most about web development is the chance to continuously learn and grow by working through challenges that arise".
For the last sentence, you can say something like "I am eager to collaborate with peers and learn from mentors, as I believe that collective knowledge fuels growth".

These are just some suggestions